.github/workflows/codeql-analysis.yml

```diff
@@ -21,6 +21,11 @@ on:
       - '**.tsx'
   schedule:
     - cron: '18 17 * * 1'
+  workflow_dispatch: # Manual
+      
+permissions:
+    actions: read
```

Comment:
Perhaps not necessary if this was working before with the default `contents: read` token?

Reply:
Switched to contents!